WebRoot/js/zimbraMail/prefs/view/ZmFilterRulesView.js

```diff
@@ -79,20 +76,17 @@ function() {
 											 context:ZmId.VIEW_FILTER_RULES});
 		this._toolbar.replaceElement(toolbarEl);
 		this._tabGroup.addMember(this._toolbar);
+		//add `ZAltButton` style to 'Create Filter' toolbar button
+		var addFilterBtn = this._toolbar.getButton(ZmOperation.ADD_FILTER_RULE);
+		addFilterBtn && addFilterBtn.delClassName('ZToolbarButton','ZButton ZAltButton'); //to look-it like ZButton
```

Comment:
I tried it initially, but in this case it was not working. Since we are creating ZmToolbar here and it takes care of creating buttons directly returned by `ZmFilterRulesController.prototype.getToolbarButtons`. This method just returns `ZmOperation` instance and not instance of `DwtButton` or `DwtControl`. 

Hence needed to follow this approach.

Thanks